src/oci.h

```diff
@@ -90,10 +90,8 @@
 
 /** Name of file containing environment variables that will be set
  * inside the VM.
- *
- * Note: name is historical.
  */
-#define CC_OCI_ENV_FILE		"/.dockerenv"
```

Comment:
by now only the name has changed,  the file is used by  container-workload.service